activerecord/test/cases/persistence_test.rb

```diff
@@ -78,6 +78,13 @@ def test_class_level_update_without_ids
     end
   end
 
+  def test_class_level_update_with_base
```

Comment:
```suggestion
  def test_update_many_with_array_of_active_record_base_objects
```

Can you also move this test up so that it directly follows `test_update_many_with_active_record_base_object`?